wasp/src/main/java/com/orhanobut/wasp/WaspHttpStack.java

```diff
@@ -11,4 +13,6 @@
 
     public void setSslSocketFactory(SSLSocketFactory sslSocketFactory);
 
+    public void setCookieHandler(CookieHandler cookieHandler);
```

Comment:
We cannot make the class package-private because for setting custom httpstack one must implement this class and pass it to Wasp.Builder.setWaspHttpStack.